README.md

```diff
@@ -0,0 +1,101 @@
+Spryker Buggregator
+=============
+
+Provide integration of [buggregator](https://buggregator.dev/) service with Spryker.
+
+Installation
+------------
+
+Require the package with composer:
+
+```
+$ composer require teufelaudio/spryker-buggregator
+```
+
+Add the `Teufelaudio` namespace to the `CORE_NAMESPACES` key in your `config/Shared/config_default.php` file:
+
+```php
+$config[KernelConstants::CORE_NAMESPACES] = [
+    // ...
+    'Teufelaudio',
+];
+```
+
+Register `RequestProfilerEventDispatcherPlugin` for Yves:
+
+```php
+<?php
+
+namespace Teufel\Yves\EventDispatcher;
```

Comment:
For example code, could we use Pyz which should be more familiar with other project teams? Same comment for the other code blocks.